javascript/javascript1/week3/NoteTakingApp/script.js

```diff
@@ -0,0 +1,31 @@
+let notes = [];
+
+function saveNote (content, id) {
+  // The saveNote function should push an object to the notes array with the keys content and id
+  notes.push({
+    content,
+    id
+  });
+}
+
+function getNote(id) {
+  for (let i = 0; i < notes.length; i++) {
```

Comment:
it works, but there's an easier way ;) 

Check this out:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find